src/cellink/widgets/spinbox.h

```diff
@@ -1,8 +1,8 @@
 #ifndef SPINBOX_H
 #define SPINBOX_H
 
-#include <QtCellink/cellink.h>
 #include <QtWidgets/qspinbox.h>
+#include "cellink.h"
```

Comment:
we might want "qtcellink/cellink.h" here, to keep the include path more clear as to what filke is being included from what module. Exocyte does something like that. 

So public headers should use a public include path...   
(might even be "<project>/<subproject>/src/<header>" in some cases, to get the module name for free without adding )

Reply:
this comment goes for all headers btw..  I think it might be part of the reason to have that as  a qt module, but I agree it's not really necessary, unless I've missed something obvious..